platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/VirtualMapBench.java

```diff
@@ -261,66 +261,4 @@ public void read() throws Exception {
 
         afterTest(true);
     }
-
```

Comment:
This class was abstract (I think it's not important to revert it)

Reply:
Agree